README.md

```diff
@@ -304,33 +323,50 @@ Jolokia env variables
 * The Jolokia env variables you can set are documented in this [document](https://github.com/jboss-openshift/cct_module/tree/master/jboss/container/jolokia/api)
 
 
-Provisioning a custom server using [Galleon](https://docs.wildfly.org/galleon/)
--------------------------------------------------------------------------------
+Including custom Galleon layers during [Galleon](https://docs.wildfly.org/galleon/) provisioning
+------------------------------------------------------------------------------------------------------------------------------------
+
+Using environment variables
+-----------------------------------------
 
-That is done during s2i build using the `GALLEON_PROVISION_LAYERS` env variable.
+That is done during s2i build by using the `GALLEON_PROVISION_LAYERS` env variable. By default, a fixed set of Galleon layers (WildFly ones and WildFly s2i specific ones) 
+can be used to compose a server. Custom Galleon layers can be incorporated in the provisioned server by setting `GALLEON_PROVISION_FEATURE_PACKS` to reference 
+Galleon feature-packs that provide Galleon layers usable with WildFly.
 
-If you want to define your own WildFly server, create a directory named `galleon` at the root of your application sources project. This directory must
-contains a provisioning.xml file. During s2i build, this file is used to provision a server.
-This server is used to replace the one present in the s2i builder image (located in $JBOSS_HOME).
+Example of a custom `mylayer` Galleon layer included into the provisioned server: `GALLEON_PROVISION_LAYERS=cloud-server,ejb-lite,jpa,mylayer` `GALLEON_PROVISION_FEATURE_PACKS=`org.example:my-galleon-pack:1.0.0.Final`
```

Comment:
maybe reword to specify that the `org.example:my-galleon-pack:1.0.0.Final` that provides the `mylayer` layer.
The relation is not explicit in the existing example

Reply:
Thanks. Fixed.